static/app/gettingStartedDocs/node/awslambda.tsx

```diff
@@ -10,71 +14,177 @@ import {
   getCrashReportModalConfigDescription,
   getCrashReportModalIntroduction,
 } from 'sentry/components/onboarding/gettingStartedDoc/utils/feedbackOnboarding';
+import {AwsLambdaArn} from 'sentry/gettingStartedDocs/node/awslambdaArnSelector';
 import {t, tct} from 'sentry/locale';
-import {trackAnalytics} from 'sentry/utils/analytics';
 import {
   getInstallConfig,
   getNodeProfilingOnboarding,
   getSdkInitSnippet,
 } from 'sentry/utils/gettingStartedDocs/node';
-import {
-  InstallationMode,
-  platformOptions,
-} from 'sentry/views/onboarding/integrationSetup';
+
+export enum ModuleFormat {
+  CJS = 'cjs',
+  ESM = 'esm',
+}
+
+export const platformOptions = {
+  moduleFormat: {
+    label: t('Module Format'),
```

Comment:
l: is this label shown anywhere? I can't see it but it would probably help establish more context